<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar adm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRegistra">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRegistra{
	background-color: rgb(66, 66, 66);
	border-radius: 2px;
	color: white;
	font: 8pt &quot;Roboto&quot;;
}

QPushButton#btnRegistra:hover{
	background-color: rgb(0, 0, 0);

}</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEmail">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineEmail{
	background-color: rgb(66, 66, 66);
	border-radius: 2px;
	color: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Roboto&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSenha">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineSenha{
	background-color: rgb(66, 66, 66);
	border-radius: 2px;
	color: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Bot Gray ADMIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Veja Tec 1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Roboto&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>Data de registro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineData">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineData{
	background-color: rgb(66, 66, 66);
	border-radius: 2px;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>dd/mm/yyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConcluido">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineExpiracao">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineExpiracao{
	background-color: rgb(66, 66, 66);
	border-radius: 2px;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Roboto&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>Data de expiração</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
